src/JsonSchema/JsonSchemaInfo.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the schemaific language
+ * governing permissions and limitations under the License.
+ */
+
+import React from 'react'
+import { Table, TBody, Tr, Td } from '../Table'
+
+export const JsonSchemaInfo = ({ schema = {} }) => {
+  const { $id, $schema } = schema
```

Comment:
Because graphql identifiers can't start with a `$` when the parliament-source-jsonschema Gatsby plugin imports the JSON schema doc into the graphql DB it changes this names to `_id` and `_schema` so we should support both identifiers here.